### ISSUE TITLE
tensorflow in python

### ISSUE THREAD

**Issue by dan881818** [tensorflow/tensorflow#39128]
1.  i tried import tnsorflow , and get alot of warning messemges 

2. after import, i runed this code : 

```
a = tf.Variable(1, name = "a")
b = tf.Variable(2, name = "b")
f = a+b
print(f)
```

but this was my put put : 
Tensor("add:0", shape=(), dtype=int32)

can someone know what is the problem ? and what to do ? 

*** i did :  pip   install --upgrade tensorflow
but it doesnt  #helped....

**Comment by Saduf2019** [comment]
@dan881818 
Could you please share the tensor flow version and share complete error log for us to analyse the issue.

**Comment by dan881818** [comment]
how can i check wich version of tensorflow i have? 
i work on jupyter via anaconda, 
on the prompt i run this code in order to install the tensorflow : 

**conda   create -n tpf3.7 python=3.7
activate tpf3.7
conda install   tensorflow
pip install --upgrade   tensorflow
import tensorflow**

**about the error messeges :** 
import tensorflow as tf
C:\Users\dansh\anaconda3\lib\site-packages\tensorflow\python\framework\dtypes.py:516: FutureWarning: Passing (type, 1) or '1type' as a synonym of type is deprecated; in a future version of numpy, it will be understood as (type, (1,)) / '(1,)type'.
  _np_qint8 = np.dtype([("qint8", np.int8, 1)])
C:\Users\dansh\anaconda3\lib\site-packages\tensorflow\python\framework\dtypes.py:517: FutureWarning: Passing (type, 1) or '1type' as a synonym of type is deprecated; in a future version of numpy, it will be understood as (type, (1,)) / '(1,)type'.
  _np_quint8 = np.dtype([("quint8", np.uint8, 1)])
C:\Users\dansh\anaconda3\lib\site-packages\tensorflow\python\framework\dtypes.py:518: FutureWarning: Passing (type, 1) or '1type' as a synonym of type is deprecated; in a future version of numpy, it will be understood as (type, (1,)) / '(1,)type'.
  _np_qint16 = np.dtype([("qint16", np.int16, 1)])
C:\Users\dansh\anaconda3\lib\site-packages\tensorflow\python\framework\dtypes.py:519: FutureWarning: Passing (type, 1) or '1type' as a synonym of type is deprecated; in a future version of numpy, it will be understood as (type, (1,)) / '(1,)type'.
  _np_quint16 = np.dtype([("quint16", np.uint16, 1)])
C:\Users\dansh\anaconda3\lib\site-packages\tensorflow\python\framework\dtypes.py:520: FutureWarning: Passing (type, 1) or '1type' as a synonym of type is deprecated; in a future version of numpy, it will be understood as (type, (1,)) / '(1,)type'.
  _np_qint32 = np.dtype([("qint32", np.int32, 1)])
C:\Users\dansh\anaconda3\lib\site-packages\tensorflow\python\framework\dtypes.py:525: FutureWarning: Passing (type, 1) or '1type' as a synonym of type is deprecated; in a future version of numpy, it will be understood as (type, (1,)) / '(1,)type'.
  np_resource = np.dtype([("resource", np.ubyte, 1)])
C:\Users\dansh\anaconda3\lib\site-packages\tensorboard\compat\tensorflow_stub\dtypes.py:541: FutureWarning: Passing (type, 1) or '1type' as a synonym of type is deprecated; in a future version of numpy, it will be understood as (type, (1,)) / '(1,)type'.
  _np_qint8 = np.dtype([("qint8", np.int8, 1)])
C:\Users\dansh\anaconda3\lib\site-packages\tensorboard\compat\tensorflow_stub\dtypes.py:542: FutureWarning: Passing (type, 1) or '1type' as a synonym of type is deprecated; in a future version of numpy, it will be understood as (type, (1,)) / '(1,)type'.
  _np_quint8 = np.dtype([("quint8", np.uint8, 1)])
C:\Users\dansh\anaconda3\lib\site-packages\tensorboard\compat\tensorflow_stub\dtypes.py:543: FutureWarning: Passing (type, 1) or '1type' as a synonym of type is deprecated; in a future version of numpy, it will be understood as (type, (1,)) / '(1,)type'.
_np_quint8 = np.dtype([("quint8", np.uint8, 1)])
C:\Users\dansh\anaconda3\lib\site-packages\tensorboard\compat\tensorflow_stub\dtypes.py:543: FutureWarning: Passing (type, 1) or '1type' as a synonym of type is deprecated; in a future version of numpy, it will be understood as (type, (1,)) / '(1,)type'.
  _np_qint16 = np.dtype([("qint16", np.int16, 1)])
C:\Users\dansh\anaconda3\lib\site-packages\tensorboard\compat\tensorflow_stub\dtypes.py:544: FutureWarning: Passing (type, 1) or '1type' as a synonym of type is deprecated; in a future version of numpy, it will be understood as (type, (1,)) / '(1,)type'.
  _np_quint16 = np.dtype([("quint16", np.uint16, 1)])
C:\Users\dansh\anaconda3\lib\site-packages\tensorboard\compat\tensorflow_stub\dtypes.py:545: FutureWarning: Passing (type, 1) or '1type' as a synonym of type is deprecated; in a future version of numpy, it will be understood as (type, (1,)) / '(1,)type'.
  _np_qint32 = np.dtype([("qint32", np.int32, 1)])
C:\Users\dansh\anaconda3\lib\site-packages\tensorboard\compat\tensorflow_stub\dtypes.py:550: FutureWarning: Passing (type, 1) or '1type' as a synonym of type is deprecated; in a future version of numpy, it will be understood as (type, (1,)) / '(1,)type'.
  np_resource = np.dtype([("resource", np.ubyte, 1)])

**Comment by mihaimaruseac** [comment]
Please fill in issue template. Please properly format error and code blocks using proper markdown syntax.

TF version can be obtained with `tf.__version__` after import, or via `pip list`. Actually, please post output of `pip list` as well as a filled in issue template.

**Comment by dan881818** [comment]
Mihai Maruseac  - did'nt understand what you mean .... what shuold i do in roder the tensorflow work via jupyter ?

**Comment by ymodak** [comment]
duplicate #31249

**Comment by dan881818** [comment]
(base) C:\Users\dansh>pip show tensorflow
Name: tensorflow
Version: 2.1.0
Summary: TensorFlow is an open source machine learning framework for everyone.
Home-page: https://www.tensorflow.org/
Author: Google Inc.
Author-email: packages@tensorflow.org
License: Apache 2.0
Location: c:\users\dansh\anaconda3\lib\site-packages
Requires: grpcio, tensorflow-estimator, scipy, wrapt, protobuf, opt-einsum, tensorboard, keras-applications, gast, keras-preprocessing, termcolor, numpy, six, google-pasta, absl-py, wheel, astor
Required-by:

(base) C:\Users\dansh>pip show numpy
Name: numpy
Version: 1.18.1
Summary: NumPy is the fundamental package for array computing with Python.
Home-page: https://www.numpy.org
Author: Travis E. Oliphant et al.
Author-email: None
License: BSD
Location: c:\users\dansh\anaconda3\lib\site-packages
Requires:
Required-by: tensorflow, tensorboard, tables, statsmodels, seaborn, scipy, scikit-learn, PyWavelets, pytest-arraydiff, patsy, pandas, opt-einsum, numexpr, numba, mkl-random, mkl-fft, matplotlib, Keras-Preprocessing, Keras-Applications, imageio, h5py, Bottleneck, bokeh, bkcharts, astropy

**Comment by dan881818** [comment]
hey,
up is my tensorflow + numpy version
Name: tensorflow
Version: 2.1.0
Name: numpy
Version: 1.18.1
what i need to do in order tensorflow worm in my jupyter ? 
tnx...

**Comment by mihaimaruseac** [comment]
Duplicate of #31249
